src/GraphQL/Execution/ExecutionHelper.cs

```diff
@@ -427,6 +460,9 @@ public static bool ShouldIncludeNode(ExecutionContext context, Directives direct
             return true;
         }
 
+        /// <summary>
+        /// TODO
```

Comment:
see http://spec.graphql.org/June2018/#DoesFragmentTypeApply()

Reply:
This method calculates the criterion for matching fragment definition (spread or inline) to a given graph type. This criterion determines the need to fill the resulting selection set with fields from such a fragment.

Reply:
```suggestion
        /// This method calculates the criterion for matching fragment definition (spread or inline) to a given graph type.
        /// This criterion determines the need to fill the resulting selection set with fields from such a fragment.
        /// <br/><br/>
        /// See http://spec.graphql.org/June2018/#DoesFragmentTypeApply()
```
Is this what you suggest?  It's still too mind-boggling for me to understand just now!

Reply:
It will fit.